src/garvis/cli.py

```
@@ -0,0 +1,125 @@
+
+    assistant = GarvisAssistant(
+        model=args.model,
+        persist_memory=not args.no_memory,
+        session_db=args.db,
+    )
+
+    prompt = " ".join(args.prompt).strip()
+    if prompt:
+        try:
+            reply = await assistant.respond(prompt, session_id=args.session)
+        except Exception as exc:  # CLI boundary: return a clear error instead of a traceback.
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)